client/src/main/java/tds/session/PauseSessionRequest.java

```diff
@@ -0,0 +1,26 @@
+package tds.session;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.UUID;
+
+/**
+ * Data for requesting to pause a {@link tds.session.Session}
+ */
+public class PauseSessionRequest {
+    private UUID browserKey;
+    private long proctorId;
+
+    public PauseSessionRequest(@JsonProperty("proctorId") long proctorId, @JsonProperty("browserKey") UUID browserKey) {
```

Comment:
I'll need to remember this technique for classes that don't have a ton of fields.